src/vorta/tray_menu.py

```diff
@@ -70,6 +72,14 @@ def set_tray_icon(self, active=False):
         """
         Use white tray icon, when on Gnome or in dark mode. Otherwise use dark icon.
         """
+
+        last_backup = EventLogModel.select().where(EventLogModel.subcommand == 'create')\
+            .where(EventLogModel.returncode == '1').where(EventLogModel.repo_url != '(NULL)')\
+            .select(pw.fn.MAX(EventLogModel.start_time)).scalar()
+        last_backup_formatted = last_backup.strftime('%d %B %H:%M')
```

Comment:
```suggestion
        last_backup = EventLogModel.select().where(EventLogModel.subcommand == 'create' & EventLogModel.returncode == '1' &
                                                   EventLogModel.repo_url != '(NULL)').select(pw.fn.MAX(EventLogModel.start_time)).scalar()
        last_backup_formatted = last_backup.strftime('%d %B %H:%M') if last_backup else self.tr("Never")   
```
You can combine multiple wheres with an &, also fixed the case where there are no successful backups

Reply:
Thank again @samuel-w ! 
it is done. This check flake8 W503 is a bug.
https://gitlab.com/pycqa/flake8/-/issues/139

Reply:
Created a PR to fix it https://github.com/borgbase/vorta/pull/706